mathsnuggets/widgets/indices.py

```diff
@@ -0,0 +1,24 @@
+import sympy
+
+from mathsnuggets.core import fields, form
+
+
+class Indices(form.Form):
+    """Indices"""
+
+    template = "Calculate `index_1` `operation` `index_2` `solution`"
+
+    index_1 = fields.Expression("index 1")
+    index_2 = fields.Expression("index 2")
```

Comment:
We'll need a special type of field that will always be displayed in standard form

Reply:
@Laurentmoutier 

- VCS/Git/**Fetch**
- Checkout the 'tom' branch
- Create a field called `StandardForm` in core/fields.py
- `StandardForm` inherits from `Expression` but converts to standard form in the `sanitize()` method:

To convert `number` in standard form:

    power = floor((sympy.log(sympy.Abs(number)) / log(10)).evalf())
    x = sympy.Mul(number, Pow(10, -power)).evalf()
    if float(x).is_integer():
        x = int(x)
    number = sympy.Mul(x, sympy.Pow(sympy.UnevaluatedExpr(10), power, evaluate=False), evaluate=False)